README.md

```diff
@@ -8,6 +8,7 @@ This repository is used for managing all our dev session resources and [rules](r
 |---------|---------|---------------|--------|
 | Rows Locking system using PostgreSQL and NodeJS | [Video](), [Slides]() | [Muhammad AbdulMoiz](https://github.com/abdulmoizeng) | 25th April, 2018
 | Typescript: beyond basics | Slides | [Abdul Qadir](https://github.com/abdulqadir93) | 21st February 2018 |
+| Named Router Outlets in Angular | [Slides](https://docs.google.com/presentation/d/1b3nPF_gFLAPS3U5H2BKkQkx34kxLZbzhX-BK6oGd3-g/edit?usp=sharing), [Code](https://github.com/recurship/dev-sessions/tree/master/sessions/named-router-outlets) | [Abdul Qadir](https://github.com/abdulqadir93) | 11th April 2018 |
```

Comment:
@abdulqadir93  Can you please move Named Router Outlets session, upwards so that it is the second talk in current list? And also, can you add links to your slides for Typescript talk? Thanks